src/utils/data.ts

```diff
@@ -20,7 +20,9 @@ export const generateDataByMonth = (originalData: any) => {
     else return 0;
   });
   const oldestMonth = orderedMonths[0];
-  const newestMonth = orderedMonths[orderedMonths.length - 1];
+  const now = new Date();
+  const newestMonth =
+    now.getFullYear() + '-' + now.getMonth().toString().padStart(2, '0');
```

Comment:
As pointed out in #568, we should use last month as `newestMonth` because if we are in 2023.1 only data of 2022.12 is available. All charts show data in last month.

The second mistake here is you might not know the truth that the value of Date.getMonth() [starts from 0](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth).